### ISSUE TITLE
[Example] I am new Research!! : [insert title]

### ISSUE THREAD

**Issue by TobiasConnerCJH** [criminaljusticehub/user-researchers#4]
# New issue for @user-researchers :pencil2:  :pencil2:  :pencil2: 

### All below tasks need to be completed before this epic can be considered completed. 

- [ ] Research Problem
- [ ] Alignment on Plan (meeting)
- [ ] Conduct Research
- [ ] Analysis of research
- [ ] Create Report 
- [ ] Report to Team
- [ ] Content Design Issue(s) Created / Further Research Issue(s) Created (PLEASE LINK BACK HERE)
